src/lists.py

```diff
@@ -42,17 +45,29 @@ def print_list(x):
 match sys.argv[1]:
     case "mean":
         # put your solution to the first exercise here
-        mean = "mean of x"
+
+        #calculate the sum 
+        sum = 0
+        for i in x:
+            sum += i
+        #calculate the mean
+        
+        mean = (sum/len(x))
```

Comment:
Excellent! And now that you know how to write code to sum all the elements in a list, I can tell you that there already is a function for this, `sum`. (You can only use it, if course, if you haven't already redefined what `sum` means as you do here).

The short version for calculating the mean of a (non-empty) list `x` would be `sum(x)/len(x)`.

Your solution is what I was looking for, I just wanted to let you know that you do not have to write this code again, as it already exists.